spotseeker_server/cache/default_cache.py

```diff
@@ -0,0 +1,44 @@
+""" Copyright 2012, 2013 UW Information Technology, University of Washington
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+"""
+from spotseeker_server.models import Spot
+
+
+def get_spot(spot_model):
```

Comment:
What are you trying to optimize with the cache?  I can't believe that going to the database to make a model is fast, but generating the JSON is slow - I would need to see a profile that has evidence of that, and then I'd still want to make sure there wasn't just a simpler solution than building a cache.

If you want a cache, the get method should take an id, not a model.